spot_rl_experiments/spot_rl/envs/skill_manager.py

```diff
@@ -119,7 +130,7 @@ def __initiate_controllers(self):
         )
         self.gaze_controller = GazeController(config=self.pick_config, spot=self.spot)
         self.place_controller = PlaceController(
-            config=self.place_config, spot=self.spot, use_policies=False
+            config=self.place_config, spot=self.spot, use_policies=use_policies
```

Comment:
unless it is described elsewhere in the code/readme, we may want to specify where these policies are located/coming from e.g., a specific config or path? If we are providing these policies, may be we can add in the readme a way to download them as well from some place just like we did for hab3.0 release.

Reply:
@tusharsangam

Reply:
@rutadesai config.yaml contains the path to individual policy weights. While setup instructions page describes how to download these weights [Like this one](https://github.com/facebookresearch/spot-sim2real/blob/main/installation/SETUP_INSTRUCTIONS.md#:~:text=%23%20Download%20weights%20(need%20only%20once)%0Agit%20clone%20https%3A//huggingface.co/spaces/jimmytyyang/spot%2Dsim2real%2Ddata%0Aunzip%20spot%2Dsim2real%2Ddata/weight/weights.zip%20%26%26%20rm%20%2Drf%20spot%2Dsim2real%2Ddata%20%26%26%20cd%20../)
However there is no individual section in readme that goes over each policy details like what are its observations, functions & output actions, etc. Is this the section you mean to add ?
Since I haven't worked on hab 3.0 can you please point me to the readme section you are referring to get more clear idea ?

Reply:
As discussed offline with @rutadesai offline, this is not needed here as info about policies is provided in the SETUP_INSTRUCTIONS.md which use will go through at the start of setup process.